### PR TITLE
Move Array#fill to "write then read" methods so the template correctly updates

### PR DESCRIPTION
**Pull request by locks**
I was using `Array#fill` to showcase how to use `TrackedArray` and the template was not updating accordingly.

### CODE REVIEW

**Comment by acorncom** [comment]
@locks might be worth adding a simple test so we don’t regress in the future?

File: test-app/tests/unit/array-test.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ const ARRAY_SETTER_METHODS = [
 // vice versa).
 expectTypeOf<TrackedArray<unknown>>().toMatchTypeOf<Array<unknown>>();
 
-module('TrackedArray', function (hooks) {
+module.only('TrackedArray', function (hooks) {
```
Comment (acorncom) [inline test-app/tests/unit/array-test.ts:52]: @locks assume we probably need this taken out too?

Comment (locks) [reply test-app/tests/unit/array-test.ts:52]: 🤦 apologies

Comment (acorncom) [reply test-app/tests/unit/array-test.ts]: @locks you're fine, just figured I could flag it for you before someone who can merge comes back 😉

**Comment by ef4** [comment]
The test failures on newer ember versions are because we need some blueprint updates. Working on that here: https://github.com/tracked-tools/tracked-built-ins/pull/409

**Comment by ef4** [comment]
If you rebase on master now tests should be passing.

**Comment by SergeAstapov** [comment]
@locks Thank you! This is now published as v3.3.0. cc @ef4

**Comment by locks** [comment]
Thank you for the quick turnaround!

**Pull request merged**